### ISSUE TITLE
Reporting missing movies that are actually on Plex

### ISSUE THREAD

**Issue by kiwitech62** [JasonHHouse/gaps#251]
Gaps is reporting multiple movies as missing that are actually on Plex and showing in collections.

I am running Plex on Windows 10, along with Radarr, Sonarr, etc. and have installed Gaps on the same server and successfully connected it to Plex. When I run a search on my main Movie library it is reporting movies that I have in Plex as missing but it reports other movies in the same collection as Owned. Both Radarr and Plex list the movies correctly, and they are named correctly for Plex.

A couple of examples are shown below:

![Gaps Screenshot](https://user-images.githubusercontent.com/64527410/130021570-3f0182bc-0333-4362-a6b5-29d933ea0cff.jpg)

Both "Escape Plan: The Extractors" and "Maze Runner: The Death Cure" are showing as Missing, but according to Plex I have both of them in the various collections:

![Plex - Escape Plan Collection](https://user-images.githubusercontent.com/64527410/130021918-7e4fcb9a-4c7d-470d-8c97-f6a7261d2195.jpg)

![Plex - Maze Runner Collection](https://user-images.githubusercontent.com/64527410/130021944-6dfeace2-04fb-4db1-929a-1fc6d7a92353.jpg)

I have many other movies I actually have being reported as Missing. I have tried removing the problem movies from Plex and then re-adding them and rescanning in Plex and re-searching in Gaps but it makes no difference.

I've linked the RSS feed generated by Gaps with the list of missing movies to Radarr as a list and it added movies that were actually missing but nothing happened regarding the existing movies that Gaps reports as Missing.

Appreciate any suggestions as to why this is happening, and what I might be able to do to fix the issue.

Server Platform:  Windows 10 Version 21H1 (OS Build 19043.1165)
Plex Server Version:  1.24.0.4930
Gaps Version:  v0.6.1
Radarr Version:  4.0.0.5219

**Comment by JasonHHouse** [comment]
The version of Gaps you're running is pretty old. Please try again with a newer version.

https://github.com/JasonHHouse/gaps/releases/tag/v0.9.7

**Comment by kiwitech62** [comment]
Thanks for that. Initially had a problem getting it to work as I had it running on my D: drive and just replaced the 'gaps.jar' file with the new version and it didn't pick up the configuration or data from the original installation so tried reconfiguring it to connect to TMDB and Plex but it wouldn't save the settings so wasn't able to scan my Plex library. 

Ended up downloading one of the older versions with the installer exe and installed using that. Found it created the installation folder in 'C:\Program Files (x86)\Gaps' with a Data sub-folder, which I didn't have on my original installation. After installing the older version I was able to configure it and connect it successfully to Plex. Ran a scan of my Movie library and it found the library and movies successfully. Updated the 'gaps.jar' file to the latest version and it recognized the database and configuration OK. Ran a search for missing movies and it successfully created a valid list. Set up a connection to the list in Radarr and it successfully added the missing movies from the list into Radarr. 

So all working fine now. Thanks and very much appreciated.
